stellarphot/core.py

```diff
@@ -240,7 +240,8 @@ class Config:
         json_encoders = {
             Quantity: lambda v: f"{v.value} {v.unit}",
             QuantityType: lambda v: f"{v.value} {v.unit}",
-            UnitType: lambda v: f"{v}",
+            Unit: lambda v: f"{v}",
```

Comment:
I assume this sets it up some you store both the units and irreducible units in JSON.   Do you know why this is more 'round-trippable'?  Is it because UnitType isn't an actual astropy object but Unit and IrreducibleUnit are?

Reply:
I think it is because the encoding is done based on the type of the setting, eg. `Unit` or `IrreducibleUnit`. I could have used just `IrreducibleUnit` here but in principle units could be more complicated, e.g. aand image data unit of `electron / sec` or something.